src/frontend/src/lib/components/send/QRCodeScan.svelte

```diff
@@ -81,3 +81,10 @@
 		{$i18n.core.text.back}
 	</button>
 </ButtonGroup>
+
+<style>
+	.qr-code-wrapper {
+		--primary-rgb: 50, 20, 105;
+		color: rgba(var(--primary-rgb), 0.6);
```

Comment:
this is needed? I had a quick test, using solely `--primary-rgb` seemed to work but maybe I missed something?